.github/workflows/pr_python.yml

```
@@ -0,0 +1,26 @@
+name: Pull request testing
```

You should name this differently otherwise it's difficult to distinguish which test on the pr page
```suggestion
name: Python Pull request testing
```